### PR TITLE
📜 Add directions on where to save custom js.

### PR DESCRIPTION
**Pull request by mxbaylee**
Added for clarity.

### CODE REVIEW

File: advanced/custom/js.mdx [hunk 1]
```diff
@@ -11,6 +11,8 @@ Custom JS allows you to add custom executable code globally. It is the equivalen
 
 ## Adding script.js
 
+To add custom JavaScript functionality globally, you need to create a `script.js` file at the root of your project. In case of a monorepo, this file should be at the same level as your `mint.json` file.
```
Comment (mayankshouche) [inline advanced/custom/js.mdx]: iirc, we'll actually pick up `.js` files nested anywhere in the content directory (path of `mint.json`) or lower

Comment (mxbaylee) [reply advanced/custom/js.mdx]: Interesting. [I just tested and this is true](https://github.com/mxbaylee/mintlify-starter/commit/a287c98f255b3f251bd5c09c280a88e80a81f316). Are we sure this is the behavior we want? 😅 


I'll update the PR.

Comment (mayankshouche) [reply advanced/custom/js.mdx]: > Are we sure this is the behavior we want? 😅

long term, probably not haha

Comment (mxbaylee) [reply advanced/custom/js.mdx]: Updated, what do you think?

Comment (mayankshouche) [reply advanced/custom/js.mdx]: lgtm!

**Review by mayankshouche — APPROVED** [review]

**Pull request merged**